s3transfer/utils.py

```diff
@@ -389,7 +389,39 @@ class NoResourcesAvailable(Exception):
     pass
 
 
-class SlidingWindowSemaphore(object):
+class TaskSemaphore(object):
+    def __init__(self, count):
+        """A semaphore for the purpose of limiting the number of tasks
+
+        :param count: The size of semaphore
+        """
+        self._semaphore = threading.Semaphore(count)
+
+    def acquire(self, tag, blocking=True):
```

Comment:
Probably worth documenting/commenting the reason there's params that are required but not used is because we want to be API compatible with the `SlidingWindowSemaphore`.